src/App.tsx

```diff
@@ -10,33 +11,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('Tab 1');
+
+  const onTabSelected = (tabTitle: string) => {
```

Comment:
```suggestion
  const handleTabSelected = (tabTitle: string) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364